index.js

```diff
@@ -21,7 +21,7 @@ let minimist = require('minimist');
 // local modules
 let Application = require('./src/browser/api/application.js').Application;
 let System = require('./src/browser/api/system.js').System;
-import { Window } from './src/browser/api/window';
+import { create } from './src/browser/api/window';
```

Comment:
Nitpick: Considering this file is as huge as it is, I think it's more descriptive to have the `Window.create` call vs just a plain `create` call. Can be hard to tell exactly what's being created.

Reply:
Trying not to import the whole file here, it helps with circular dependencies on the typescript conversion

Reply:
Ah, good call. Maybe hit it with an `import { create as createWindow} from './src/browser/api/window';` ?